src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -285,7 +286,11 @@ void addLoad(Load load, LfNetworkParameters parameters) {
     }
 
     void addLccConverterStation(LccConverterStation lccCs, LfNetworkParameters parameters) {
-        getOrCreateLfLoad(null, parameters).add(lccCs, parameters);
+        if (!LfHvdcUtils.isHvdcDandlingInIdm(lccCs, null)) {
```

Comment:
isHvdcDandlingInIidm (missing i)
same everywhere in the code

Reply:
Faiit